routes/auth/token.js

```diff
@@ -0,0 +1,71 @@
+const express = require("express");
+const jwt = require("jsonwebtoken");
+
+const { User } = require("../../model/User");
+const {
+  generateAccessToken,
+  generateRefreshToken,
+} = require("../../utils/generateToken");
+
+const router = express.Router();
+
+router.get("/validate", async (req, res, next) => {
+  try {
+    const deliOrderToken = req.headers["authorization"]?.split(" ")[1];
+
+    jwt.verify(deliOrderToken, process.env.JWT_SECRET_KEY);
+    return res.status(200).json({ message: "정상인증된 유저입니다." });
+  } catch {
+    return res.status(401).json({ error: "유효하지 않은 유저입니다." });
+  }
+});
+
+router.post("/refresh", async (req, res, next) => {
+  try {
+    const { deliOrderUserId: userId, deliOrderRefreshToken: userRefreshToken } =
+      req.body;
+
+    if (!userId) {
+      return res.status(401).json({ error: "유효하지 않은 유저입니다." });
+    }
+
+    const existUser = await User.findById(userId).lean();
+    const dbRefreshToken = existUser.deliOrderRefreshToken;
```

Comment:
이 부분에서 구조분해 할당으로 한번에 받을 수 있는데 이렇게 쓰신 이유라도 있으신가여?😁
```suggestion
    const { deliOrderRefreshToken: dbRefreshToken } = await User.findById(userId).lean();
```

Reply:
개인적인 코드스타일로 차례대로 선언한 뒤 읽어가는게 가독성이 더 좋다고 느껴져서 선호합니다!